src/app/mat-main-nav/desktop-menu/desktop-menu.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, OnInit, ViewEncapsulation } from "@angular/core";
+import { menuItems } from "../menu-items.const";
+import { IMenuItem } from "../menu-item.interface";
+import { environment } from "@environment";
+import { MatMenuTrigger } from "@angular/material/menu";
+
+@Component({
+  selector: "app-desktop-menu",
+  templateUrl: "./desktop-menu.component.html",
+  styleUrls: ["./desktop-menu.component.scss"],
+  encapsulation: ViewEncapsulation.None
+})
+export class DesktopMenuComponent implements OnInit {
+  menuItems$: IMenuItem[] = menuItems;
+  hostURI: string = environment.adminsUIHostUri;
+  private currentMenuTrigger: MatMenuTrigger;
+  private currentMenuElement: Element;
+
+  constructor() {}
+
+  ngOnInit(): void {}
+  /**
+   * TO:DO investigate mouseover functionality
+   * add (mouseover)="menuOver(menuTrigger)" (mouseout)="menuOut(menuTrigger)"
+   * add [hasBackdrop]="false" to sub menu
+   * @param e
+   */
+  menuOver(e: MatMenuTrigger): void {
+    if (this.currentMenuTrigger !== e) {
+      if (!this.currentMenuTrigger) {
+        this.currentMenuTrigger = e;
+      } else {
+        this.currentMenuTrigger.closeMenu();
+        this.currentMenuTrigger = e;
+      }
+      e.openMenu();
+    } else {
+      if (e.menuClosed) {
+        e.openMenu();
+      }
+    }
+    this.currentMenuElement = document.querySelector(`#${e.menu.panelId}`);
+  }
+  menuOut(e: MatMenuTrigger): void {}
```

Comment:
empty method needed?

Reply:
Will remove all methods. Experimental but not actually material ethos. People have asked for desktop hover behaviour. But they closed the PR